### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Microsoft IIS Home Page_Keyboard: Focus is not visible on "Hide Sidebar" and "Skip This" buttons.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#55]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Issue: Keyboard Focus Visibility

**MAS ID Violated**: MAS2.4.7

**Repro Steps:**
1. Open "Microsoft IIS" Home Page using the URL: https://manage.iis.net/
2. Tab till ""Hide Sidebar" and "Skip This" button and validate that keyboard focus is clearly visible on it.

**Actual Result:**
Keyboard focus is not visible on "Hide Sidebar" and "Skip This" buttons.

**Expected Result:**
When "Hide Sidebar" and "Skip This" button receives keyboard focus, a dotted border should appear around these controls so that user can differentiate clearly between the focused and non-focused controls on the page.

**User Impact**
This Success Criterion helps anyone who relies on the keyboard to operate the page, by letting them visually determine the component on which keyboard operations will interact at any point in time.
People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment:**
[Bug 55_Focus is not visible on Hide Sidebar and Skip This buttons.pptx](https://github.com/Microsoft/IIS.WebManager/files/2317148/Bug.55_Focus.is.not.visible.on.Hide.Sidebar.and.Skip.This.buttons.pptx)

**Comment by accessibility18** [comment]
[Bug 55_Focus is not visible on Hide Sidebar and Skip This buttons.pptx](https://github.com/Microsoft/IIS.WebManager/files/2314274/Bug.55_Focus.is.not.visible.on.Hide.Sidebar.and.Skip.This.buttons.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;#A11yMAS;
